ONSdigital/eq-questionnaire-runner
# New routing rules for greater then and less than numbers

### What is the context of this PR?
This PR is to add new schemas for greater than and less than numbers using the new routing rules engine.

Also:
- Fixes some typos
- Removes a duplicate routing test operators test
- Removes duplicate path finder test for old schema
- Adds tests for greater than and less than, testing for them to fail when number is equal
- Removes functional tests for non single condition routing

### How to review 
- Tests pass
- Check test changes make sense
- Check new schemas work as expected

### Checklist
